File: Java/FizzBuzz/FizzBuzz.java
public class FizzBuzz {
	public String fizzBuzz(int number) {

		String returnString;
		returnString = String.valueOf(number);

		if ((number % 3) == 0) {
			returnString = "Fizz";
			if ((number % 5) == 0)
				returnString += "Buzz";
		} else if ((number % 5) == 0)
			returnString = "Buzz";

		return returnString;
	}

	public String fizzBuzz(
		int number,
		String multiple3Word,
		String multiple5Word,
		String multiple15Word) {

		String returnString;
		returnString = String.valueOf(number);

		if ((number % 3) == 0) {
			returnString = multiple3Word;
			if ((number % 5) == 0)
				returnString = multiple15Word;
		} else if ((number % 5) == 0)
			returnString = multiple5Word;

		return returnString;
		}
}

File: WebFundamentals/Week1/Day5-JS/FizzBuzz.js
for (i = 0; i <= 100; i++) {
    let three = i % 3 == 0;
    let five = i % 5 == 0;
    if (three)
        if (five)
            console.log(i + ": FizzBuzz");
        else
            console.log(i + ": Fizz");
    else
        console.log(i);

}